_data/simple-icons.json

```diff
@@ -5370,6 +5370,11 @@
             "hex": "DF0067",
             "source": "https://www.prestashop.com/en/media-kit"
         },
+        {
+            "title": "Presto",
+            "hex": "0F2648",
```

Comment:
Looks good to me, however the 'most used' colour in the logo found [here](https://github.com/prestodb/presto/blob/414ab2a6bbdcca6479c2615b048920adac34dd20/presto-docs/src/main/resources/logo/web/main/black/FB_Presto_Logo_BlackBG-01.svg) is `#5890FF`, would you mind updating to that?

Reply:
They seem to use loads of different shades of blue with no clear preference for one so I suppose that's as good a choice as any. Updated.